.github/workflows/release.yml

```diff
@@ -67,41 +68,46 @@ jobs:
           args: "--locked --release"
           strip: true
 
-      - name: Prepare binary
-        run: |
-          mkdir -p /tmp/artifact/
-          cp "target/${{ matrix.platform.target }}/release/${{ matrix.platform.bin }}" "/tmp/artifact/${{ matrix.platform.file-name }}"
-
-      - name: Calculate sha256sum
-        run: sha256sum /tmp/artifact/${{ matrix.platform.file-name }} >> /tmp/artifact/${{ matrix.platform.file-name }}.sha256
-
       - name: Upload binary
         uses: actions/upload-artifact@v3
         with:
           name: ${{ matrix.platform.file-name }}
-          path: /tmp/artifact/${{ matrix.platform.file-name }}
-
-      - name: Upload SHA256
-        uses: actions/upload-artifact@v3
-        with:
-          name: ${{ matrix.platform.file-name }}.sha256
-          path: /tmp/artifact/${{ matrix.platform.file-name }}.sha256
+          path: "target/${{ matrix.platform.target }}/release/${{ matrix.platform.bin }}"
 
   finalize-release:
     name: Upload artifacts, trigger homebrew workflow
     needs: [build-artifacts, validate-version]
-    if: github.event_name == 'release'
     runs-on: ubuntu-latest
     permissions:
       contents: write
     steps:
       - name: Download am artifacts
         uses: actions/download-artifact@v3
         with:
-          name: build-artifacts
+          path: artifacts
+
+      - name: Prepare files
+        run: |
+          ls -R
```

Comment:
why

Reply:
Was useful for debugging. I will remove it.